docs/adguard-for-linux/vpn-cli-interaction.md

```diff
@@ -0,0 +1,24 @@
+---
+title: AdGuard VPN CLI interaction
+sidebar_position: 5
+---
+
+## AdGuard VPN CLI interaction
+
+Using AdGuard CLI in **automatic proxy mode** together with AdGuard VPN CLI in **tunnel mode** is impossible due to a
+routing loop: AdGuard VPN CLI's traffic currently can not be excluded from being transparently proxied by AdGuard CLI,
+and AdGuard CLI's traffic must go through the AdGuard VPN CLI's tunnel.
+
+These are the two possible solutions:
+
+1. Switch AdGuard CLI to **manual proxy mode**.
```

Comment:
I think it is better to use subheaders here, because plain numbered list more looks like a steps, not like alternatives solutions.